code/modules/mob/living/carbon/human/life.dm

```diff
@@ -338,45 +308,13 @@
 
 		if(null_internals) //something wants internals gone
 			internal = null //so do it
-
+			update_action_buttons_icon()
 
 	if(internal) //check for hud updates every time this is called
-		update_internals_hud_icon(1)

```

Comment:
Likewise, we don't need to update the action buttons of internals every time you breathe to make the icon green; if it wasn't green to begin with, someone messed up.